app/models/annotation/AnnotationService.scala

```diff
@@ -185,18 +222,77 @@ class AnnotationService @Inject()(
                 resolutionRestrictions =
                   annotationLayerParameters.resolutionRestrictions.getOrElse(ResolutionRestrictions.empty)
               )
-              volumeTracingId <- client.saveVolumeTracing(volumeTracing)
+              volumeTracingAdapted = oldPrecedenceLayerProperties.map { p =>
+                volumeTracing.copy(
+                  editPosition = p.editPosition,
+                  editRotation = p.editRotation,
+                  zoomLevel = p.zoomLevel,
+                  userBoundingBoxes = p.userBoundingBoxes
+                )
+              }.getOrElse(volumeTracing)
+              volumeTracingId <- client.saveVolumeTracing(volumeTracingAdapted)
             } yield volumeTracingId
           case _ =>
             Fox.failure(s"Unknown AnnotationLayerType: ${annotationLayerParameters.typ}")
         }
       } yield AnnotationLayer(tracingId, annotationLayerParameters.typ, annotationLayerParameters.name)
-    }
+
+    def fetchOldPrecedenceLayer: Fox[Option[FetchedAnnotationLayer]] =
+      if (existingAnnotationLayers.isEmpty) Fox.successful(None)
+      else
+        for {
+          oldPrecedenceLayer <- selectLayerWithPrecedence(existingAnnotationLayers)
+          tracingStoreClient <- tracingStoreService.clientFor(dataSet)
+          oldPrecedenceLayerFetched <- if (oldPrecedenceLayer.typ == AnnotationLayerType.Skeleton)
+            tracingStoreClient.getSkeletonTracing(oldPrecedenceLayer, None)
+          else tracingStoreClient.getVolumeTracing(oldPrecedenceLayer, None, skipVolumeData = true)
+        } yield Some(oldPrecedenceLayerFetched)
+
+    def extractPrecedenceProperties(oldPrecedenceLayer: FetchedAnnotationLayer): PrecedenceTracingProperties =
+      oldPrecedenceLayer.tracing match {
+        case Left(s) =>
+          PrecedenceTracingProperties(
+            s.editPosition,
+            s.editRotation,
+            s.zoomLevel,
+            s.userBoundingBoxes ++ s.userBoundingBox.map(
+              com.scalableminds.webknossos.datastore.geometry.NamedBoundingBox(0, None, None, None, _))
+          )
+        case Right(v) =>
+          PrecedenceTracingProperties(
+            v.editPosition,
+            v.editRotation,
+            v.zoomLevel,
+            v.userBoundingBoxes ++ v.userBoundingBox.map(
+              com.scalableminds.webknossos.datastore.geometry.NamedBoundingBox(0, None, None, None, _))
+          )
+      }
+
+    for {
+      oldPrecedenceLayer <- fetchOldPrecedenceLayer
+      precedenceProperties = oldPrecedenceLayer.map(extractPrecedenceProperties)
+      newAnnotationLayers <- Fox.serialCombined(allAnnotationLayerParameters)(p =>
+        createAndSaveAnnotationLayer(p, precedenceProperties))
```

Comment:
Please add a comment explaining why and when this might be necessary (possibly changing precedence to the new annotation, therefore precedenceproperties need to be copied to the new layer), that the history might be lost in this case, and that it is done precautionarily since it's not clear yet if the new layer get's precedence, since it depends on the id which is not know yet. (Tried to rephrase my new-won understanding, thanks a lot for walking me through!)

PS: I find the name PrecedenceProperties quite confusing, since they don't define the precedence, but rather AnnotationProperties that are stored on the AnnotationLayer and selected by some precedence^^